config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -167,6 +167,39 @@ presubmits:
       testgrid-tab-name: pr-secrets-store-csi-driver-e2e-azure
       description: "Run e2e test with azure provider for Secrets Store CSI driver."
       testgrid-num-columns-recent: '30'
+  - name: pull-secrets-store-csi-driver-e2e-provider
+    decorate: true
+    decoration_config:
+      timeout: 25m
+    always_run: true
+    optional: true # change to false once the provider job is stable
+    path_alias: sigs.k8s.io/secrets-store-csi-driver
+    branches:
+    - master
+    labels:
+      # this is required because we want to run kind in docker
+      preset-dind-enabled: "true"
+      # this is required to make CNI installation to succeed for kind
+      preset-kind-volume-mounts: "true"
+      # sets up the azure keyvault parameters used for testing
+      preset-azure-secrets-store-creds: "true"
```

Comment:
should the fake provider have azure creds?

Reply:
fixed